### PR TITLE
Extend countDecimals() for negative values

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Extend countDecimals() for negative values
- Change some comments and names 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1208

### CODE REVIEW

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

File: src/wallet.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { Accounts, Account, TransactionBody, V3Keystore, V3KeystoreOptions } fro
 import Ain from './ain';
 import { validateMnemonic, mnemonicToSeedSync } from 'bip39';
 import Reference from './ain-db/ref';
-const AIN_HD_DERIVATION_PATH = "m/44'/412'/0'/0/"; /* default wallet address for AIN */
-const MAX_TRANSFERABLE_DECIMALS = 6; /* The maximum transferable decimal places of values */
+const AIN_HD_DERIVATION_PATH = "m/44'/412'/0'/0/";  // The hardware wallet derivation path of AIN
```
Comment (shyun-comcom) [inline src/wallet.ts:5]: 저희 AIN wallet이나 aFan wallet에서는 이제 `m/44'/60'/0'/0/`을 derivation path prefix로 사용하기로 했는데, 여기서는 이 path로 유지하는게 맞을까요, 아니면 변경하는게 맞을까요?

Comment (platfowner) [reply src/wallet.ts:6]: Right, just added a todo.
One problem is that we already have some v3 wallet files.
Thanks!

**Comment by platfowner** [comment]
Thanks for the review! @shyun-comcom @woomurf Now merging..
Please let me know if you have any other comments.

**Pull request merged**